koku/providers/aws/provider.py

```diff
@@ -136,13 +136,14 @@ def cost_usage_source_is_reachable(self, credentials, data_source):
             internal_message = f"Unable to access account resources with ARN {credential_name}."
             raise serializers.ValidationError(error_obj(key, internal_message))
 
-        s3_exists = _check_s3_access(storage_resource_name, creds)
+        region_name = data_source.get("bucket_region")
```

Comment:
Should we give this a default of `"us-east-1"`?
```suggestion
        region_name = data_source.get("bucket_region") or "us-east-1"
```

Reply:
I wasn't sure of the difference between a region of `None` and and explicit region when passed to `boto3.resource()`. We are effectively passing `None` currently, so I just kept it that way to avoid unexpected changes in behavior.